### ISSUE TITLE
The New Fluent Theme should be Fluent2

### ISSUE THREAD

**Issue by robert-abeo** [dotnet/wpf#9561]
### Description

I've brought this up before but this is a last attempt to get the naming right. Once it releases I suspect we are stuck. I won't rehash fully why we should be using the official name: Fluent2, however, it's a matter of branding and versioning.

 1. If the versioning is out of sync this is going to cause an issue if ever we do need a Fluent1 or a future Fluent3
 2. You are violating Microsoft's own branding and naming conventions. See here:
     * https://fluent2.microsoft.design/
     * https://www.figma.com/community/file/836828295772957889/microsoft-fluent-2-web

For non-developers calling it Fluent is fine. But developers and designers need to be more specific.

### Reproduction Steps

N/A

### Expected behavior

Fluent2 should be the name of the new theme

### Actual behavior

Fluent without any version is the current name of the theme

### Known Workarounds

Fluent Theme

**Comment by miloush** [comment]
I agree it is in the spirit of existing themes that Fluent2 should be able to co-exist with any other future theme. This also provides a stability guarantee (not migrating apps automatically to a new theme if they explicitly ask for Fluent)

**Comment by h3xds1nz** [comment]
I fully agree with this and support it, there's no reason why we should divert from the official name for the current theme.
